eth_typing/abi.py

```diff
@@ -0,0 +1,4 @@
+from typing import Union
+
+TypeStr = str
+Decodable = Union[bytes, bytearray]
```

Comment:
I agree with the direction you went in chat: moving this module to an external library and adding the dependency.